### ISSUE TITLE
tool chain: integrate 'ali2dep' tool

### ISSUE THREAD

**Issue by cproc** [genodelabs/genode#3361]
I'd like to integrate the 'ali2dep' tool (issue #3144) with the 19.05 tool chain, but I'm not sure what exactly needs to be done.

So far, I checked out the 'ali2dep' repository and typed 'make', which printed the following error message:

```
build.gpr:5:08: undefined attribute "create_missing_dirs"
gprbuild: "build/build" processing failed
Makefile:16: recipe for target 'all' failed
make: *** [all] Error 4
```

@senier, @jklmnn: can you help here?

**Comment by jklmnn** [comment]
You need the GNAT Community Toolchain to use the `create_missing_dirs` directive in gpr files. If you only use the Debian GNAT you can just create a directory named `build` in the repository and comment the line in the gpr file out.

**Comment by cproc** [comment]
I commented the line out and now the following error occurs:

```
gnatgcc -c -gnatwI osint.adb
osint.adb:2597:17: "Null_FD" is undefined
```

This is on Ubuntu 16.04 with GNAT 5.4.0.

**Comment by jklmnn** [comment]
I'm afraid we don't support anything older that GNAT 6.3.0 (which doesn't support the `create_missing_dirs` directive. Anything newer than that should work also.

**Comment by cproc** [comment]
Ok, then I'll try to build it with the bootstrap tool chain which is built by the 'tool_chain' script.

**Comment by cproc** [comment]
Fixed in c6c352a.

With GCC 6.3.0 the same 'Null_FD' error occured, so this commit depends on #3307.
